ReactAndroid/src/main/java/com/facebook/react/ReactRootView.java

```diff
@@ -200,6 +203,21 @@ public boolean onTouchEvent(MotionEvent ev) {
     return true;
   }
 
+  @Override
+  public boolean dispatchKeyEvent(KeyEvent ev) {
+    if (mReactInstanceManager == null || !mIsAttachedToInstance ||
+      mReactInstanceManager.getCurrentReactContext() == null) {
+      FLog.w(
+        ReactConstants.TAG,
+        "Unable to handle key event as the catalyst instance has not been attached");
+      return super.dispatchKeyEvent(ev);
+    }
+    ReactContext reactContext = mReactInstanceManager.getCurrentReactContext();
+    DeviceEventManagerModule.RCTDeviceEventEmitter eventEmitter = reactContext.getJSModule(DeviceEventManagerModule.RCTDeviceEventEmitter.class);
+    mAndroidTVRootViewHelper.handleKeyEvent(ev, eventEmitter);
+    return super.dispatchKeyEvent(ev);
```

Comment:
should we return true here? similarly to what we do with touches we can pretty safely assume the key press event is going to be consumed by the react app. Calling into super will just default to returning false.
This is only relevant to apps that embed react native root views, but if we both handle the key and return false the "master" app may also pass the same key event to some other view which is undesirable